commit.py

```
@@ -9,133 +9,191 @@
-        for chunk in iter(lambda: f.read(65536), b""):
-            hasher.update(chunk)
-        hashed_file = hasher.hexdigest()
-except Exception as e:
+def hash_current_docx_binary():
+    try:
+        with open(path, "rb") as f:
+                hasher = hashlib.sha256()
+                for chunk in iter(lambda: f.read(65536), b""):
+                    hasher.update(chunk)
+                hashed_file = hasher.hexdigest()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)